neurom/sections.py

```diff
@@ -92,6 +92,20 @@ def area(section):
     return _aggregate_segments(section, mm.segment_area)
 
 
+def point_at_path_fraction(section, fraction):
+    '''Computes the point which corresponds to the fraction
+    of the path length along the piecewise linear curve which
+    is constructed from the section points.
+
+    Args:
+        0, 1, 2 correspoding to 3D cartesian coordinates
+
+    Returns:
+        The 3D coordinates of the aforementioned point
+    '''
+    return mm.path_fraction_point(tuple(n.value[:3] for n in section), fraction)
```

Comment:
Do you really need to slide here? I think you can just pass the points as values.